MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,368 @@
+</td>
+<td>
+Specifies enum type for the data field.
+</td>
+</tr>
+</table>
+
+{% tabs %}
+{% highlight MinLength attribute}
+
+    [MinLength(5,ErrorMessage ="Password length must be greater than 5 characters")]
+    public string Password { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight MaxLength attribute}
+
+    [MaxLength(20,ErrorMessage ="Maximum password length should be less than 20")]
+    public string Password { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight Required attribute}
+
+    [Required(AllowEmptyStrings =false, ErrorMessage ="Name is required")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight StringLength attribute}
+
+    [StringLength(20,ErrorMessage ="Name should not exceed 20 characters")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+{% tabs %}
+{% highlight EnumDataType attribute}
+
+    [EnumDataType(typeof(Gender), ErrorMessage = "Please select Gender")]
+    public Gender Gender { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Bindable attribute
+
+It specifies whether the field should be auto generated or not. If bindable set to false the field will not be auto generated. 
+
+{% tabs %}
+{% highlight Bindable attribute}
+
+    [Bindable(false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Editable attribute
+
+It specifies whether the data field is editable or not.
+
+{% tabs %}
+{% highlight Editable attribute}
+
+    [Editable(false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## ReadOnly attribute
+
+It specifies whether the data field is read only or not.
+
+{% tabs %}
+{% highlight ReadOnly attribute}
+
+    [ReadOnly(false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+Note: `ReadOnlyAttribute` takes higher priority than `EditableAttribute`
+
+## DataType attribute
+
+It specifies data type for the field.
+
+Supported data types are Text, MultilineText, Date, DateTime and Time.
+
+{% tabs %}
+{% highlight DataType attribute}
+
+    [DataType(DataType.MultilineText)]
+    public string Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Custom attribute
+The data form supports the following custom attribute, and these attributes can be accessible using `Syncfusion.Maui.DataForm`.
+
+### DataFormDisplayOptions attribute
+
+<table>
+<tr>
+<th>
+Property
+</th>
+<th>
+Details
+</th>
+</tr>
+<tr>
+<td>
+[RowSpan]
+</td>
+<td>
+Specifies the row span for the data form item.
+</td>
+</tr>
+<tr>
+<td>
+[ColumnSpan]
+</td>
+<td>
+Specifies the column span for the data form item.
+</td>
+</tr>
+<tr>
+<td>
+[ValidMessage]
+</td>
+<td>
+Specifies positive message to be shown when validation is passed. 
+</td>
+</tr>
+<tr>
+<td>
+[ShowLabel]
+</td>
+<td>
+Specifies whether the label should be visible or not.
+</td>
+</tr>
+<tr>
```

Comment:
RowOrder property missed. Mention above row order. It should be above ItemsOrderInRow

Reply:
row order inserted